programs/relayer/src/modules/eth_to_cosmos.rs

```diff
@@ -175,9 +175,17 @@ impl RelayerService for EthToCosmosRelayerModuleService {
             cosmos_events.len()
         );
 
+        // Extract the inner events from the EurekaEventWithHeight, as our relay_events takes Vec<EurekaEvent>
+        let eth_events_inner = eth_events.into_iter().map(|e| e.event).collect();
+        let cosmos_events_inner = cosmos_events.into_iter().map(|e| e.event).collect();
+
         let tx = self
             .tx_builder
-            .relay_events(eth_events, cosmos_events, inner_req.target_client_id)
+            .relay_events(
+                eth_events_inner,
+                cosmos_events_inner,
+                inner_req.target_client_id,
+            )
```

Comment:
why do we strip the height here.

Reply:
lol, yeah, forgot to fix it all the way down, had this in just to keep things working. Fixing this too